src/compat/db_value_printer.cpp

```diff
@@ -656,3 +656,30 @@ db_sprint_value (const db_value *value, string_buffer &sb)
   db_value_printer printer (sb);
   printer.describe_value (value);
 }
+
+#ifndef NDEBUG
+void
+db_value_print_console (const db_value *value, bool add_newline, char *fmt, ...)
```

Comment:
Is there anything I can see in more detail if I run the function below instead of the existing db_fprint_value function?
```
db_fprint_value (stdout , &value)
```

Reply:
It's not to see more details, but because I want to show it by adding other messages I want to the output.
I'm trying to leave something like where it was printed during debugging.